ERCS/erc-7633.md

```diff
@@ -0,0 +1,202 @@
+---
+eip: 7633
```

Comment:
```suggestion
eip: 7634
```
Numbers are assigned by ERC editors/associates. Please don't self assign.
To avoid any perception of number sniping, I am assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.